src/components/Navbar/Navigation.css

```diff
@@ -0,0 +1,34 @@
+.nav{
+    display: flex;
+    background-color: #000000;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    width:100%;
+    box-shadow: 0 0 5px 1px gray;
+}
+.logo{
+    display: flex;
+    align-items: center;
+    padding-left: 40px;
+}
+.logo-text{
+    padding-left: 5px;
+    font:28px 'Roboto';
+    color: white;
+}
+.auth{
+    display: flex;
+    align-items: center;
+    padding-right: 40px;
+}
+.auth-btn{
+    padding: 10px 16px;
+    border-radius: 5px;
+    margin-right: 15px;
+    border:none;
+}
+.theme-icon{
+    font-size:24px;
+    color:white;
```

Comment:
Suggestion : Use variable names for colors, IMO it makes it easy when debugging to see where you used which color.